cmake/custom_cxxstandard.cmake

```diff
@@ -55,14 +55,13 @@ elseif (CMAKE_CXX_COMPILER_ID MATCHES AppleClang)
     endif()
 
 elseif (CMAKE_CXX_COMPILER_ID MATCHES Clang)
-    if (CMAKE_CXX_COMPILER_VERSION VERSION_LESS 3.6)
-        message(FATAL_ERROR "CLANG version must be at least 3.6!")
+    if (CMAKE_CXX_COMPILER_VERSION VERSION_LESS 4.0)
+        message(FATAL_ERROR "CLANG version must be at least 4.0!")
```

Comment:
Should I set it to 5 then? At least the structured bindings which I'm using should be covered by 4. What do you think?

Reply:
Thanks, gtk on structured bindings as the key feature. I'm ok with leaving as 4 for now, then. I'll keep in mind that this value doesn't get exercised since we don't have a min Clang lane on Azure. Conda's running clang 12 & 13, fwiw.

Reply:
Let's go for Clang 5 then 👍